diesel_cli/tests/print_schema.rs

```diff
@@ -390,3 +390,80 @@ r"pub mod custom_schema {
 }
 ");
 }
+
+#[test]
+#[cfg(feature = "postgres")]
+fn print_schema_with_foreign_keys() {
+    let p = project("print_schema_with_foreign_keys").build();
+    let db = database(&p.database_url());
+
+    // Make sure the project is setup
```

Comment:
Can't help but notice the redundant comment that is also spelt wrong (should be "is set up") which was successfully cargo culted into every single test case 😆